lib/features/registration/bloc/registration_bloc.dart

```diff
@@ -80,7 +80,7 @@ class RegistrationBloc extends Bloc<RegistrationEvent, RegistrationState> {
 
       await authRepositoy.registerUser(
         tempUser: tempUser,
-        isTempUser: false,
+        isNewUser: false,
```

Comment:
Does this change also work for the normal registration flow? Based on the name of the method I guess its used in both places

Reply:
Its just a name of variable change, doesnt change functionality